test/controllers/organizations/home_controller_test.rb

```diff
@@ -12,7 +13,14 @@ class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
       get home_index_path(script_name: "/#{@organization.slug}")
 
       assert_response :success
-      assert_select "title", text: "#{@organization.name} | \"Pet Rescue\""
+      assert_select "title", text: "#{ActsAsTenant.current_tenant.name} | \"Pet Rescue\""
```

Comment:
This is failing. Although we assert specific dom elements in this test suite, I am recommending we steer away from it as it couples the test suite too tightly with the dom and leads to test failures whenever we touch the dom in the future. 

I personally don't think we need to test this particular feature, but if we wanted to test something, I would just test the index action on the controller returns a collection of 4 pets that belong to the organization set in the test context. Or, just remove the test changes altogether.